common/src/main/java/xyz/wagyourtail/jsmacros/client/api/classes/Draw3D.java

```diff
@@ -241,54 +251,210 @@ public Box addPoint(PositionCommon.Pos3D point, double radius, int color) {
     /**
      * Draws a cube({@link Box}) with a specific radius({@code side length = 2*radius})
      *
-     * @param x      x value of the center point
-     * @param y      y value of the center point
-     * @param z      z value of the center point
+     * @param x x value of the center point
+     * @param y y value of the center point
+     * @param z z value of the center point
      * @param radius 1/2 of the side length of the cube
-     * @param color  point color
+     * @param color point color
+     *
      * @return the {@link Box} generated, and visualized
+     *
      * @see Draw3D.Box
      * @since 1.4.0
      */
     public Box addPoint(double x, double y, double z, double radius, int color) {
-        return addBox(x - radius, y - radius, z - radius, x + radius, y + radius, z + radius, color, color, true, false);
+        return addBox(
+            x - radius,
+            y - radius,
+            z - radius,
+            x + radius,
+            y + radius,
+            z + radius,
+            color,
+            color,
+            true,
+            false
+        );
     }
 
     /**
      * Draws a cube({@link Box}) with a specific radius({@code side length = 2*radius})
      *
-     * @param x      x value of the center point
-     * @param y      y value of the center point
-     * @param z      z value of the center point
+     * @param x x value of the center point
+     * @param y y value of the center point
+     * @param z z value of the center point
      * @param radius 1/2 of the side length of the cube
-     * @param color  point color
-     * @param alpha  alpha of the point
-     * @param cull   whether to cull the point or not
+     * @param color point color
+     * @param alpha alpha of the point
+     * @param cull whether to cull the point or not
+     *
      * @return the {@link Box} generated, and visualized
+     *
      * @see Draw3D.Box
      * @since 1.4.0
      */
     public Box addPoint(double x, double y, double z, double radius, int color, int alpha, boolean cull) {
-        return addBox(x - radius, y - radius, z - radius, x + radius, y + radius, z + radius, color, color, alpha, alpha, true, cull);
+        return addBox(
+            x - radius,
+            y - radius,
+            z - radius,
+            x + radius,
+            y + radius,
+            z + radius,
+            color,
+            color,
+            alpha,
+            alpha,
+            true,
+            cull
+        );
+    }
+
+    /**
+     * @param x top left
+     * @param y
+     * @param z
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z) {
```

Comment:
Perhaps these should return Surface? Otherwise surface-specific methods cannot be used by the caller (affects typescript output)

NB: Multipart suggestion, accept or reject all of them together
```suggestion
    public Surface addDraw2D(double x, double y, double z) {
```